server/repository/src/db_diesel/sensor_row.rs

```diff
@@ -43,7 +43,15 @@ pub enum SensorType {
     Laird,
 }
 
-#[derive(Clone, Queryable, Insertable, AsChangeset, Debug, PartialEq)]
+pub fn get_sensor_type(serial: &String) -> SensorType {
+    match serial.split('|').nth(1) {
+        Some("BLUE_MAESTRO") => SensorType::BlueMaestro,
+        Some("LAIRD") => SensorType::Laird,
+        _ => SensorType::BlueMaestro,
+    }
+}
```

Comment:
Think it doesn't really fit into the repository layer but if I understood correctly this is temporary anyway?

Reply:
I'd lifted it out the existing sync translation, since it is required in two places now. 
Service layer? where would I best put it?

Reply:
Hmm, Adrian is adding some sensor stuff in service layer, maybe a TODO for now is ok and we can relocate to service when we see where sensor things live in service layer ?